src/extension.ts

```diff
@@ -106,6 +107,28 @@ export function activate(context: vscode.ExtensionContext) {
   );
   context.subscriptions.push(indexes);
 
+  let oeRuntimes: Array<any>;
+  let defaultRuntime;
+  let defaultDLC;
+
+  oeRuntimes = vscode.workspace
+    .getConfiguration("abl.configuration")
+    .get<Array<any>>("runtimes")!;
+  if (oeRuntimes.length === 0) {
+    vscode.window.showWarningMessage(
+      "No OpenEdge runtime configured on this machine"
+    );
+  }
+
+  if (oeRuntimes.length === 1) {
```

Comment:
simplify this code, try to avoid using let variable.

`const defaultRuntime = oeRuntimes.length === 1 ? oeRuntimes[0] : oeRuntimes.find((runtime) => runtime.default);
`